packages/sources/coinpaprika/src/endpoint/crypto.ts

```diff
@@ -179,3 +128,61 @@ export const execute: ExecuteWithConfig<Config> = async (request, _, config) =>
     batchablePropertyPath,
   )
 }
+
+const getConvertedCoins = async (
+  jobRunID: string,
+  base: string | string[],
+  inputOverrides: OverrideObj,
+  context: AdapterContext,
+): Promise<RequestedCoins> => {
+  const overrider = new Overrider(internalOverrides, inputOverrides, AdapterName, jobRunID)
+  const [overriddenCoins, remainingSyms] = overrider.performOverrides(base)
+  let requestedCoins = overriddenCoins
+  if (remainingSyms.length > 0) {
+    const coinsResponse = await getCoinIds(context, jobRunID)
+    requestedCoins = Overrider.convertRemainingSymbolsToIds(
+      overriddenCoins,
+      remainingSyms.map((sym) => sym.toUpperCase()),
+      coinsResponse,
+    )
+  }
+  return requestedCoins
+}
+
+const handleBatchedRequest = (
```

Comment:
In the future keeping this in its original place will make for easier review, as the diff is lost.

Reply:
Noted